tests/sentry/tasks/test_weekly_escalating_forecast.py

```diff
@@ -61,9 +73,10 @@ def test_single_group_escalating_forecast(self, mock_query_groups_past_counts):
         )
 
         run_escalating_forecast()
-        group_forecast = GroupForecast.objects.all()
-        assert len(group_forecast) == 1
-        assert group_forecast[0].group == group_list[0]
+        fetched_forecast = EscalatingGroupForecast.fetch(group_list[0].project.id, group_list[0].id)
+        assert fetched_forecast.project_id == group_list[0].project.id
+        assert fetched_forecast.group_id == group_list[0].id
+        assert fetched_forecast.forecast == FORECAST_LIST_MOCK
```

Comment:
The forecast is `[200] * 14` because I didn't add a lot of data from the past week in the tests, and it defaults to `200` when this happens (see [here](https://github.com/getsentry/sentry/blob/master/src/sentry/issues/escalating_issues_alg.py#L62-L63)).  I figured the tests for the algorithm itself should suffice to actually test the forecast values

Reply:
Okay. This is low end forecast. Thank you for clarifying.